src/kyc.ts

```diff
@@ -48,7 +48,10 @@ export const personalDataAndDocumentsKycSchema = z.object(
     }),
     phoneNumber: z.string(),
     selfieDocument: z.string(),
-    identificationDocument: z.string(),
+    // identificationDocument: z.string(),
```

Comment:
I can understand the need for requiring a front and back picture for an identification document, however, as it stands currently, this would introduce a breaking change to the current schema. Current consumers of this schema _could_ in theory just use `identificationDocumentFront` instead of `identificationDocument` and ignore the two other new optional fields, but it would still be a breaking change. If this is required, we can think about how to add support for this into the spec without introducing a breaking change.

Reply:
Hi @jophish , its required. How can we do that without introducing a breaking change? One way is just add a identificationDocumentBack and idNumber and left identificationDocumment without changes.